docs/content/en/usage.md

```diff
@@ -24,20 +24,22 @@ See [ky](https://github.com/sindresorhus/ky#options) docs for all available opti
 **Example: GET JSON data**
 
 ```js
-const res = await $http.get('https://unpkg.com/nuxt/package.json')
-const data = await res.json()
+const data = await $http.$get('https://unpkg.com/nuxt/package.json')
 ```
 
-Alternatively for json only you can use `$` prefixed shortcut that smartly parses response using [destr](https://github.com/nuxt-contrib/destr).
+In most of the case, you want to get the JSON response. You can use `$` prefixed shortcut that smartly parses response using [destr](https://github.com/nuxt-contrib/destr).
```

Comment:
One thing is that destr will return text if it cannot be parsed as JSON, so it is safe to use $get even for non-JSON response too :)

Reply:
I added a warning alert to precise it 👍 thanks for this precision 😀